viewing_party/party.py

```diff
@@ -1,23 +1,248 @@
+        for i in range(len(user_data["watched"])):
+            watched_genre_list.append(user_data["watched"][i]["genre"])
+        
+        most_watched_count = 0
+        most_watched_genre = ''
+
+        # iterate through genre list, counting how many times each genre appears
+        for genre in watched_genre_list:
+            genre_count = watched_genre_list.count(genre)
+            # update most_watched_genre and genre count with whatever comes back highest
+            if genre_count > most_watched_count:
+                most_watched_count = genre_count
+                most_watched_genre = genre
+
+        return most_watched_genre
+    else:
+        return None
+        
 
 # -----------------------------------------
 # ------------- WAVE 3 --------------------
 # -----------------------------------------
+def get_unique_watched(user_data):
+    """
+    Creates list of movies that user has watched but friends haven't.
+
+    Parameters: user data, including user's watched list, and friends watched lists
+
+    Returns: new list of movie information dictionaries
+    """
+    # create empty list to store unique movies
+    user_unique_watched = []
+
+    # iterate over user's watched list
+    for i in range(len(user_data["watched"])):
+        # set flag to signal if we're still looking
+        searching_for_movie = True
+
+        # set current title we're looking for
+        looking_for_title = user_data["watched"][i]["title"]
+
+        while searching_for_movie:
+            # iterate through friends list
+            for friend in user_data["friends"]:
+                friend_index = user_data["friends"].index(friend)
+                # iterate through every movie on friend's watch list
+                if searching_for_movie:
+                    for num in range(len(user_data["friends"][friend_index]["watched"])):
+                        # if title matches title we're looking for, break out of loop and start looking for next title
+                        # once a title has been found once, we don't need to see if it's been watched by multiple friends
+                        if user_data["friends"][friend_index]["watched"][num]["title"] == looking_for_title:
+                            searching_for_movie = False
+                            break
+                        else:
+                            continue
+                # if no longer searching for movie, break out of loops until searching movie is changed
+                elif not searching_for_movie:
+                    break
+            # if after iterating through all friends watched lists we are still looking, add movie to the unique list
+            if searching_for_movie:
+                user_unique_watched.append(user_data["watched"][i])
+                # then change flag to switch to next movie
+                searching_for_movie = False
+        # continue until all movies on user's watch list have been checked. 
+        continue

```

Comment:
We generally use `continue` if we need to leave a loop iteration early to go to the next one. When we place `continue` at the end of a loop with no statements after it, it is the same as if we did not call `continue`, so we should omit it.